src/crypto/crypto_util.cc

```diff
@@ -14,6 +14,12 @@
 
 #include "math.h"
 
+#ifdef OPENSSL_FIPS
+#if OPENSSL_VERSION_MAJOR >= 3
```

Comment:
I think we might need a similar guard in `crypto_util.h`. I'm getting a compilation error: 
```
In file included from ../src/crypto/crypto_keys.h:6:0,
                 from ../src/crypto/crypto_cipher.h:6,
                 from ../src/crypto/crypto_aes.h:6,
                 from ../src/node_crypto.h:31,
                 from ../src/crypto/crypto_bio.h:27,
                 from ../src/crypto/crypto_bio.cc:22:
../src/crypto/crypto_util.h:28:12: fatal error: openssl/fips.h: No such file or directory
 #  include <openssl/fips.h>
            ^~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
Sorry about that, I had `fips.h` in /usr/include/openssl and did not notice it. I've added a guard for it now which I hope works. Thanks